recipe/meta.yaml

```diff
@@ -18,11 +18,11 @@ requirements:
     build:
         - python  # [win]
         - cmake
-        - libtiff 4.0.6
+        - libtiff 4.0.*
```

Comment:
The other `libtiff` available is `4.0.2`. Not sure if this is OK as I had issues with using `libtiff 4.0.2` from defaults on OS X.